src/riak_kv_ts_error_msgs.hrl

```diff
@@ -87,3 +87,9 @@
         ["The '", ParamName, "' parameter is part the primary key, and must have an ",
          "equals clause in the query but the ", atom_to_list(Op), " operator was used."])
 ).
+
+-define(
+   E_TOO_MANY_SUBQUERIES(N),
```

Comment:
Why have this as a macro instead of a function? Looks like it's only called in one place anyway.